### ISSUE TITLE
H2O not listed in Wkikipedia

### ISSUE THREAD

**Issue by proyb6** [h2o/h2o#1731]
It’s strange to find H2O is missing from the comparison table, should we add them?

https://en.m.wikipedia.org/wiki/Comparison_of_web_server_software

**Comment by desertnaut** [comment]
Why should it be listed in _Web server_ software??

**Comment by utrenkner** [comment]
@desertnaut Are you sure, you are in the right project? According to the title of *this* project, h2o is "an optimized HTTP server with support for HTTP/1.x and HTTP/2". Thus, it would fit extremely well under "web server software"...

Maybe you mistook this project for the similarly named https://github.com/h2oai ?

**Comment by proyb6** [comment]
@desertnaut It's a H2O is a web server and I have been using for some time on my VPS. :)
I feel it's a good idea to consider about renaming H2O because finding tutorials is a bit of challenge.

**Comment by csmk** [comment]
> should we add them?

I don't think you need the developer's permissions to add it. Actually, it's better if done by independent people/source. So go for it!

**Comment by desertnaut** [comment]
@utrenkner you are damn right I am in the wrong project! I thought I was in the machine learning-related H2O indeed!

Apologies guys (I could delete the comment, but I guess it will make a nice funny story, so I 'm leaving it)...
